### ISSUE TITLE
[Feature Request] Prevent fast-forwarding from desyncing emulated RTC from system clock

### ISSUE THREAD

**Issue by autofire372** [citra-emu/citra#5627]
I was playing Pokémon Omega Ruby, spent much of the time in fast-forward, and found that by the time I was done (about 4:45 PM EST), the game believed it was almost 9:30 at night. The game was synced with my computer when I started, and I had the RTC set to "System Clock", so my only conclusion is that fast-forwarding accelerated the emulated RTC. 

Citra version: Canary 1886 | HEAD-6f8ea3c (2020-11-24)

**Comment by xperia64** [comment]
Yes, the system time/fixed clock defines the guest 3DS clock at the start of emulation, and it speeds up or slows down based on emulation speed.

Other emulators have the option to always keep the clocked synced. It's probably good to have as an option, but it may break things when enabled.

**Comment by JohelEGP** [comment]
> Other emulators

Including citra-android.

**Comment by JohelEGP** [comment]
Actually, it's just that its description is misleading. It doesn't reflect the device's clock, but its start.
![Screenshot_20210407-203232_Trebuchet](https://user-images.githubusercontent.com/21071787/113950983-9ef11400-97e0-11eb-9f51-f24dc2784ec2.png)
